workspacehelper/tfc_variable.go

```diff
@@ -212,6 +215,7 @@ func (t *TerraformCloudClient) UpdateTerraformVariables(variables []*tfc.Variabl
 			Key:       &v.Key,
 			Value:     &v.Value,
 			HCL:       &v.HCL,
+			Category:  &v.Category,
```

Comment:
This is the main change. The rest is only to support the latest `go-tfe` version.